tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/Microsoft.DotNet.SourceBuild.Tasks.csproj

```diff
@@ -2,15 +2,16 @@
   <PropertyGroup>
     <TargetFramework>netstandard1.5</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <CoreSetupPlatformAbstractionsPath>../../../src/core-setup/src/managed/Microsoft.DotNet.PlatformAbstractions/</CoreSetupPlatformAbstractionsPath>
```

Comment:
Please use `$(ProjectDir)src/core-setup/src/managed/Microsoft.DotNet.PlatformAbstractions/` That way we don't easily break the relative paths by moving this project.